JetNews/app/src/main/java/com/example/jetnews/ui/article/ArticleScreen.kt

```diff
@@ -65,28 +64,34 @@ fun ArticleScreen(postId: String) {
         }
     }
 
-    Column {
-        TopAppBar(
-            title = {
-                Text(
-                    text = "Published in: ${post.publication?.name}",
-                    style = MaterialTheme.typography().subtitle2
-                )
-            },
-            navigationIcon = {
-                VectorImageButton(R.drawable.ic_back) {
-                    navigateTo(Screen.Home)
+    Scaffold(
+        topAppBar = {
+            TopAppBar(
+                title = {
+                    Text(
+                        text = "Published in: ${post.publication?.name}",
+                        style = MaterialTheme.typography().subtitle2
```

Comment:
Do we need to override this style? Is the default not good enough?

Reply:
This defaults to H6 which doesn't fit on one line. Lets leave it for now and do a design review after this PR.

Reply:
SG Thanks